### DISCUSSION TITLE
哔哩哔哩下载器 视频大于一个G就无法下载了 可否放开限制？

### DISCUSSION THREAD

**Discussion by 15201269733** [the1812/Bilibili-Evolved#1606] [功能建议 / Ideas]
在此先感谢开发者的努力，您极大的帮助了我。

关于视频大于一个G就判定为无法下载，我觉得非常不方便，尤其是下载电影的时候，大多数电影的1080p画质都大于一个G，大部分1080p高码率的番剧都大于一个G。

我很需要这些高清的片源，因此我希望可以放开限制，可以警告超过一个G可能会导致下载失败，但是用户可以选择是否继续下载超过一个G的文件。

我觉得这样做更稳妥，因为很多用户的网络质量很好。
![1031EB7A-9987-4918-89C4-17544F4385A8](https://user-images.githubusercontent.com/79733654/109373663-2fd3f600-78eb-11eb-88e3-b546f02ac4bb.png)

**Comment by the1812** [comment]
1. 不是无法下载, 下面导出的各种方式都能下, 需要你主动去学习使用
2. 也不是因为网络质量才限制的, 而是以前很多人不看提示就乱下, 下出问题了又觉得脚本不正常, 我就一刀切了
